cmd/controller/controller.go

```diff
@@ -61,6 +61,10 @@ type Controller struct {
 	oldGCBehavior bool // feature flag to revert to old behavior where we delete the secrets instead of relying on owners reference.
 }
 
+// Annotation flag key that the controller will check on the found Secret
+// objects to decide if it's will take over and manage it or not.
+const ManagedAnnotation = "sealedsecrets.bitnami.com/managed"
```

Comment:
Let's avoid hard-coding the annotation prefix. Instead, let's export the annotation prefix from https://github.com/bitnami-labs/sealed-secrets/blob/42b5c465bf75edc4f9003b23b07b66ceda5e2db0/pkg/apis/sealed-secrets/v1alpha1/types.go#L15 and use it here